piezo_web_app/PiezoWebApp/src/services/application_builder/template_populator.py

```diff
@@ -0,0 +1,70 @@
+class TemplatePopulator:
+
+    def build_template(self, validated_arguments):
```

Comment:
Am I right in thinking that `validated_arguments` should be a dictionary? If so, could we please rename it `validated_arguments_dict`? (Or perhaps `validated_args_dict` for short)

Reply:
On further reflection, `inputs` or `parameters` may be a better word than `arguments`, as the latter could be confused with arguments for an individual method.